subprojects/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/EclipseModelBuilder.java

```diff
@@ -196,54 +198,88 @@ private void populate(Project project) {
             }
         }
 
+        // org.gradle.plugins.ide.eclipse.model.NonRenamableProject  takes
```

Comment:
Please remove all these inline comments.

Reply:
*smack head* Whoops sorry, it was very late.

Interesting observation though: "Project" and "EclipseProject" are very overloaded terms around these parts :-) you can see I was struggling to remember which was which, esp. when variable names often clash with different classes too...